Hold to activate save/load states

strabitz opened the pull request:
Add a timer to avoid accidental save/load states. Hold to activate instead. A lockout timer is also present so that holding won't cause a million save/load states.

AlexanderHarrison commented:
- Looks like you removed the loop over potential humans and only get port 1. This causes other ports to fail.
- I think the save threshold should be greatly shortened or removed. It is generally much more destructive to accidentally save a state than it is to accidentally load a state. And people will be loading states much more than saving them, making them wait 1/3 a second each time is a bit much. I'd say remove it.

strabitz commented:
I agree, I'll revert the changes to the load state and have it only apply to save state. I'll also revert the change I made to cause the port issue.

strabitz commented:
@AlexanderHarrison this should be good to go now. Removed the delay from loading savestates, and also fixed the port bug, which was actually on the previous stable build as well
